src/renderer.cpp

```diff
@@ -490,12 +492,54 @@ void gl_renderer::draw_moby_models(
 			glDisableVertexAttribArray(5);
 		}
 	}
+
+	if (show_bounding_box) {
+		glm::vec3 max = model.bounding_box.max / (float) INT16_MAX, min = model.bounding_box.min / (float) INT16_MAX;
+
+		std::vector<float> bounding_box_verts {
+			min.x, min.y, min.z, max.x, max.y, min.z, max.x, min.y,  min.z,
+			min.x, max.y, min.z, max.x, max.y, min.z, min.x, min.y, min.z,
+			min.x, min.y, max.z, max.x, max.y, max.z, max.x, min.y,  max.z,
+			min.x, max.y, max.z, max.x, max.y, max.z, min.x, min.y, max.z,
+			min.x, max.y, min.z, min.x, max.y, max.z, max.x, max.y,  min.z,
+			min.x, max.y, max.z, max.x, max.y, max.z, max.x, max.y, min.z,
+			min.x, min.y, min.z, min.x, min.y, max.z, max.x, min.y,  min.z,
+			min.x, min.y, max.z, max.x, min.y, max.z, max.x, min.y, min.z,
+			min.x, min.y, min.z, min.x, max.y, min.z, min.x, min.y, max.z,
+			min.x, max.y, min.z, min.x, max.y, max.z, min.x, min.y, max.z,
+			max.x, min.y, min.z, max.x, max.y, min.z, max.x, min.y,  max.z,
+			max.x, max.y, min.z, max.x, max.y, max.z, max.x, min.y, max.z
+		};
+
+		glm::vec4 colour = glm::vec4(1.0f, 1.0f, 1.0f, 1.0f);
+		GLuint vertex_buffer;
+
+		glPolygonMode(GL_FRONT_AND_BACK, GL_LINE);
+		glUseProgram(shaders.solid_colour_batch.id());
+
+		glBindBuffer(GL_ARRAY_BUFFER, local_to_world_buffer);
+
+		glGenBuffers(4, &vertex_buffer);
```

Comment:
This looks like it's uploading buffers to the GPU every frame and then never deleting them. I'd recommend you generate the buffer once and then store its handle as a member of the moby_model class using the gl_buffer struct I made: https://github.com/chaoticgd/wrench/blob/5e138a671505b2c36dec37e4de7fc0955e5dd3cd/src/gl_includes.h#L27-L35

This will ensure that resources are managed properly e.g. if the model object is moved it will zero the old handle so it doesn't call glDeleteBuffers on it twice.